plugins/eventMacro/eventMacro/FileParser.pm

```diff
@@ -113,6 +111,19 @@ sub parseMacroFile {
 				} else {
 					undef %block
 				}
+				
+				
+			} elsif ($_ =~ /call [^{]/) {
```

Comment:
Perl syntax: `$_ =~` is unnecessary here, this can be written as `} elsif (/call [^{]/) {`.

I don't understand why this is here. It matches two cases: `call ^` and `call {`, and in either case, the text before and after is ignored. So this will match `notacall ^wat`, which probably isn't intentional?

The `call {` case completely prevents the following branch (`} elsif ($_ eq "call {") {`) from ever triggering. If that is intentional, we can remove the following branch.